src/model/immutable/DraftBlockMap.js

```diff
@@ -0,0 +1,17 @@
+/**
+ * Copyright (c) 2013-present, Facebook, Inc.
+ * All rights reserved.
+ *
+ * This source code is licensed under the BSD-style license found in the
+ * LICENSE file in the root directory of this source tree. An additional grant
+ * of patent rights can be found in the PATENTS file in the same directory.
+ *
+ * @providesModule DraftBlockMap
+ * @flow
+ */
+
+'use strict';
+
+import type Immutable from 'immutable';
+
+export type DraftBlockMap = Immutable.OrderedSet<string>;
```

Comment:
This type seems wrong. Isn't this more like a `Map<string, DraftBlockRenderConfig>`?